src/ballet/chacha20/test_fd_chacha20_rng.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include "fd_chacha20.h"
+
+#define TEST_KEY_SIZE FD_CHACHA20_KEY_SIZE
+#define TEST_NONCE_SIZE FD_CHACHA20_NONCE_SIZE
+
+/* Test fd_chacha20_rng_init function */
+void test_fd_chacha20_rng_init()
+{
+  unsigned char key[TEST_KEY_SIZE];
+  unsigned char nonce[TEST_NONCE_SIZE];
+
+  /* Generate key and nonce using /dev/urandom */
+  FILE *fp = fopen("/dev/urandom", "rb");
+  fread(key, sizeof(key), 1, fp);
+  fread(nonce, sizeof(nonce), 1, fp);
+  fclose(fp);
```

Comment:
Why add a dependency to urandom when you don't actually care about the randomness? I'd suggest initializing it to something you know (like all 1s) or using `fd_rng` with a fixed seed.

Reply:
totally agree, just removed it ✅ using a test vector instead